anghammarad-client-node/README.md

```diff
@@ -0,0 +1,40 @@
+# Anghammarad Node Client
+
+[Anghammarad](https://github.com/guardian/anghammarad) is a notification service for the Guardian's dev teams.
+
+## Usage
+
+To use Anghammarad from your project, include its client library in
+your package.json.
+
+`yarn add @guardian/anghammarad` or `npm install --save @guardian/anghammarad`
+
+The Anghammarad client contains a function that will send a notification.
+
+```js
+import { Anghammarad } from '@guardian/anghammarad';
+
+const client = new Anghammarad();
+client.notify({ ...parameters });
+```
```

Comment:
An example or two might be helpful to readers as well, especially with the optional SNS client? I realise the same observation applies to the Scala client documentation right now, but no need to hold the JS docs down to the existing (lack of) quality :-)

This is a minor point, no dramas!